utils/add_platform_rule.py

```diff
@@ -359,7 +369,7 @@ def main():
     create_parser.add_argument(
         '--group', default="", help='The group directory of the rule to create.')
     create_parser.add_argument(
-        '--name', required=True, help='The name of the Kubernetes object to check. Required.')
+        '--name', help='The name of the Kubernetes object to check. Required.')
```

Comment:
Do we also remove `Required` from `help`

Reply:
Thanks! Done